gum/backend-linux/gumprocess-linux.c

```diff
@@ -404,6 +404,7 @@ gum_process_modify_thread (GumThreadId thread_id,
   if (thread_id == gum_process_get_current_thread_id ())
   {
 #ifndef HAVE_ANDROID
+#ifndef HAVE_MUSLLIBC
```

Comment:
```suggestion
#if !defined (HAVE_ANDROID) && !defined (HAVE_MUSLLIBC)
```

Reply:
Changed to:
#if !defined (HAVE_ANDROID) && (defined (__GLIC__) || defined (__UCLIBC__))

and accompanied by a comment.